.mailmap

```diff
@@ -0,0 +1,2 @@
+Jörn Zaefferer <joern.zaefferer@gmail.com>
+Richard D. Worth <rdworth@gmail.com>
```

Comment:
Oh ok, i get what you mean but i can't see a red empty sign :no_mouth: , anyways no issue, in my next commit i will end files with newline.